sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BeamSqlUnparseContext.java

```diff
@@ -69,6 +91,10 @@ public SqlNode toSql(RexProgram program, RexNode rex) {
     return super.toSql(program, rex);
   }
 
+  private static String escapeForZetaSql(String input) {
```

Comment:
nit: this function doesn't provide much value, just inline this?

Reply:
Done.